### DISCUSSION TITLE
Bootstrap 5 compatability with Internet Explorer

### DISCUSSION THREAD

**Discussion by kushgup123** [twbs/bootstrap#39144] [Q&A] [answered]
My team is looking to migrate to Bootstrap version 5 from version 3. But the issue is that some of our users are still on Internet Explorer. I wanted to understand what kind of compatibility issues are faced by the users when they run a web app whose frontend has bootstrap 5 code. Can you please attach pictures as well

**Comment by mdo** [comment, marked as answer]
Per https://getbootstrap.com/docs/5.3/getting-started/browsers-devices/, we don't support it at all. You'll face layout issues and likely other issues. Not 100% sure what else because we just don't test it. v4 was the last major version to support it.
